src/databricks/sql/client.py

```diff
@@ -501,6 +524,13 @@ def execute(
         """
         if parameters is None:
             parameters = []
+
+        elif not Connection.server_parameterized_queries_enabled(
+            self.connection.protocol_version
+        ):
+            raise Error(
+                "Parameterized operations are not supported by this server. DBR 14.1 is required."
```

Comment:
Our plan is to re-enable the code path for inlining parameters in a subsequent PR, yes?

Reply:
Yep!